openapi/paths/credit-memos@{id}@timeline.yaml

```
@@ -0,0 +1,68 @@
+parameters:
+  - $ref: ../components/parameters/resourceId.yaml
+get:
+  x-products:
+    - Core
+  tags:
+    - Credit memos Timeline
+  summary: Retrieve a list of credit memo timeline messages
+  operationId: GetCreditMemoTimelineCollection
+  x-sdk-operation-name: getTimelineCollection
+  description: |
+    Retrieve a list of credit memo timeline messages.
+  parameters:
+    - $ref: ../components/parameters/collectionLimit.yaml
+    - $ref: ../components/parameters/collectionOffset.yaml
+    - $ref: ../components/parameters/collectionFilter.yaml
+    - $ref: ../components/parameters/collectionSort.yaml
+    - $ref: ../components/parameters/collectionQuery.yaml
+  responses:
+    '200':
+      description: A list of credit memo timeline messages was retrieved successfully.
+      headers:
+        Pagination-Total:
+          $ref: ../components/headers/Pagination-Total.yaml
+        Pagination-Limit:
+          $ref: ../components/headers/Pagination-Limit.yaml
+        Pagination-Offset:
+          $ref: ../components/headers/Pagination-Offset.yaml
+      content:
+        application/json:
+          schema:
+            type: array
+            items:
+              $ref: ../components/schemas/Timelines/CreditMemoTimeline.yaml
+    '401':
+      $ref: ../components/responses/Unauthorized.yaml
+    '403':
+      $ref: ../components/responses/Forbidden.yaml
+post:
+  x-products:
+    - Core
+  tags:
+    - Credit memos Timeline
+  summary: Create a credit memo timeline message.
+  operationId: PostCreditMemoTimeline
+  x-sdk-operation-name: createTimelineMessage
```

Suggestion:
All other timelines are using `createTimelineComment` instead of  `createTimelineMessage`. Should we keep it consistent?
```suggestion
  x-sdk-operation-name: createTimelineComment
```